library/CMakeLists.txt

```diff
@@ -273,6 +279,7 @@ package_set_postinst_prerm(
     "hiprand;rocrand"
     "${CMAKE_INSTALL_PREFIX}/hiprand/lib;${CMAKE_INSTALL_PREFIX}/rocrand/lib"
     "${CMAKE_INSTALL_PREFIX}/hiprand/include;${CMAKE_INSTALL_PREFIX}/rocrand/include"
+    "/opt/rocm/hiprand/lib;/opt/rocm/rocrand/lib"
```

Comment:
This does not make a package relocatable. We should not hard code this to /opt/rocm/

Reply:
Same comment as above.